### PR TITLE
chore(examples): add create-next-app example

### LINKED ISSUE #212: An example of a Next.JS
# Feature request 🚀

### Expected:
 An example of how to get started on with next.js
There may be some compatibility issues to deal with

### PR DESCRIPTION
**Pull request by antoniojps**
# PR Checklist

- [x] Fix linting errors
- [x] Label has been added

## Change information
Resolves #212

### CODE REVIEW

File: examples/create-next-app/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import Head from 'next/head'
+import { Text, Card, Note, Code, Spacer } from '@zeit-ui/react'
+
+export default function Home() {
+  return (
+    <div className="container">
```
Comment (unix) [inline examples/create-next-app/pages/index.js:6]: Maybe we can try `Page` component here?

Comment (antoniojps) [reply examples/create-next-app/pages/index.js:6]: Sure, that was created by create-next-app, just pushed a commit with this change.

**Comment by unix** [comment]
@antoniojps 
I didn't make it clear, I mean `zeit-ui/react` has a component called [Page](https://react.zeit-ui.co/en-us/components/page), we can omit a lot of  CSS code if we use `Page` component.

**Comment by antoniojps** [comment]
> @antoniojps
> I didn't make it clear, I mean `zeit-ui/react` has a component called [Page](https://react.zeit-ui.co/en-us/components/page), we can omit a lot of CSS code if we use `Page` component.

Yes, I was going to that and forgot.

I didn't use any of the `Page.Header`, `Page.Footer`, etc since that might fall out of the scope of a simple example.

**Pull request merged**